horreum-web/src/domain/tests/AllTests.tsx

```diff
@@ -387,8 +367,8 @@ export default function AllTests() {
                     })
                     const move = useMoveToFolder({
                         name: arg.row.original.name,
-                        folder: folder || "",
-                        onMove: (id, newFolder) => updateFolder(id, folder || "", newFolder, alerting).then(loadTests),
+                        folder: pagination.folder || DEFAULT_FOLDER,
```

Comment:
changing to `pagination.folder` means we need to add `pagination` to the dependencies for the `useMemo` otherwise the hook might run with stale values of `pagination`

Reply:
Think I set the folder property as a dependency on pagination [here](https://github.com/whitingjr/Horreum/blob/fix-pagination-test-list/horreum-web/src/domain/tests/AllTests.tsx#L401). Is that as you expected or something else ?